records_mover/db/redshift/redshift_db_driver.py

```diff
@@ -51,6 +53,7 @@ def schema_sql(self, schema: str, table: str) -> str:
         else:
             return out
 
+    # TODO: this should be in loader class
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker